room/settings.md

```diff
@@ -6,8 +6,97 @@ menubar: room_menubar
+- **Max. Dauer (4)** Hier können Sie die maximale Dauer des Meetings in Minuten festlegen. Die festgelegte Dauer startet, sobald sie das Meeting starten. Ist die Zeit abgelaufen, wird das Meeting automatisch beendet.
+
+{% include notification.html status="is-success" icon="fas fa-info-circle" message="**Mehr erfahren**  [Welche Raumarten gibt es?](../room/types)" %}
+<br>
+{% include notification.html message="**Wichtig!** Wenn Sie Einstellungen geändert haben, speichern Sie diese mit einem Klick auf den grünen Speichern-Button unten rechts" %}
+
+<hr>
+## Sicherheit
+
+![Einstellungen Sicherheit]({{ site.baseurl }}/assets/img/screenshots/room/settings/sicherheit.png#center)
+
+- **Zugangscode (1)** Mit einem Klick auf das Würfelsymbol können Sie einen neuen zufällig generierten Zugangscode erstellen. Wenn Sie den Raum ohne Zugangscode zur Verfügung stellen möchten, klicken Sie auf das Mülleimer Symbol. Es erscheint der Schriftzug "-- ungeschützt --"
+- **Gäste zulassen (2)** Standardmäßig ist die Option "Gäste zulassen" deaktiviert. So können nur angemeldete Nutzer den Raum betreten. Möchten Sie den Raum auch für nicht-angemeldete Nutzer verfügbar machen, so aktivieren Sie diese Option.
+- **Neue Mitglieder zulassen (3)** Wenn Sie diese Option aktivieren, können Personen sich selbst für den Raum als Mitglied anmelden. Um Mitglied des Raumes zu werden, ist der Zugangscode notwendig (Vorausgesetzt Sie haben Ihren Raum mit einem Zugangscode versehen). Wenn diese Option nicht aktiviert ist, müssen Sie Mitglieder manuell zu dem Raum hinzufügen.
+
+{% include notification.html message="**Wichtig!** Wenn Sie Einstellungen geändert haben, speichern Sie diese mit einem Klick auf den grünen Speichern-Button unten rechts" %}
+
+<hr>
+## Teilnehmer
+
+![Einstellungen Teilnehmer]({{ site.baseurl }}/assets/img/screenshots/room/settings/teilnehmer.png#center)
+
+- **Max. Teilnehmerzahl (1)** Legen Sie die maximale Teilnehmerzahl des Raumes fest. Standardmäßig haben Räume keine Teilnehmerbegrenzung. 
+- **Standardrolle (2)** Hier können Sie die Rolle einstellen, die ein Mitglied beim betreten des Raumes erhalten soll. Dies funktioniert nur für angemeldete Nutzer, nicht für Gäste. Als Standardrolle ist "Teilnehmer" ausgewählt. Wählen Sie "Moderator" aus, um jedem neuen Mitglied des Raumes die Moderatorenrolle zuzuweisen. Sie können die Rollen für jedes Mitglied bei Bedarf auch nachträglich anpassen (siehe [Mitglieder verwalten](../room/features/membership#mitglieder-verwalten)). Mehr zu den Berechtigungen der unterschiedlichen Rollen finden Sie unter [Mitgliedschaftsrollen](../room/features/membership#mitgliedschaftsrollen)
+- **Warteraum (3)** Hier können Sie die Warteraumfunktion aktivieren. Diese ist Standardmäßig deaktiviert. Wenn Sie die Funktion für Mitglieder (also registrierte Nutzer) aktivieren möchten, klicken Sie auf "Aktiviert". Wenn auch Gäste den Warteraum nutzen sollen, klicken Sie auf "Aktiviert für Gäste". 
```

Comment:
- Deaktiviert: Alle kommen sofort in den Raum
- Aktiviert: Alle Personen die keine Moderatoren (inkl. Raumeigentümer und Co-Eigentümer) kommen in den Warteraum
- Aktiviert für Gäste: Alle angemeldeten Nutzer kommen sofort in den Raum, nur Gäste müssen im Warteraum warten